server/settings/editorial_creators.py

```diff
@@ -3,6 +3,86 @@
     SettingsField,
     task_types_enum,
 )
+from ayon_server.exceptions import BadRequestException
+from pydantic import validator
+
+
+product_type_enum = [
```

Comment:
These constants variable nemase should be uppered `product_type_enum` -> `PRODUCT_TYPE_ENUM` etc.